Patch-Source: https://github.com/AyatanaIndicators/ayatana-indicator-display/commit/c8ab74b0ad048003dd5c920353442e12994514b2

From 3aea0a0ad5991f1c00b233e5adf84091211bd0d0 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Mon, 6 Dec 2021 23:00:37 +0100
Subject: [PATCH 3/3] Fix wrong Exec= in .desktop file

Previously it was pointing to the path
  /usr/libexec/ayatana-indicator-display-service
instead of
  /usr/libexec/ayatana-indicator-display/ayatana-indicator-display-service

Align the implementation with ayatana-indicator-datetime
---
 data/CMakeLists.txt                       | 1 +
 data/ayatana-indicator-display.desktop.in | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/data/CMakeLists.txt b/data/CMakeLists.txt
index cd052b6..f023e8c 100644
--- a/data/CMakeLists.txt
+++ b/data/CMakeLists.txt
@@ -45,6 +45,7 @@ endif()
 ##
 
 # build it
+set (pkglibexecdir "${CMAKE_INSTALL_FULL_PKGLIBEXECDIR}")
 configure_file ("${CMAKE_CURRENT_SOURCE_DIR}/${CMAKE_PROJECT_NAME}.desktop.in" "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_PROJECT_NAME}.desktop")
 
 # install it
diff --git a/data/ayatana-indicator-display.desktop.in b/data/ayatana-indicator-display.desktop.in
index d51920d..941e850 100644
--- a/data/ayatana-indicator-display.desktop.in
+++ b/data/ayatana-indicator-display.desktop.in
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Type=Application
 Name=Ayatana Indicator Display
-Exec=@CMAKE_INSTALL_FULL_LIBEXECDIR@/ayatana-indicator-display-service
+Exec=@pkglibexecdir@/ayatana-indicator-display-service
 OnlyShowIn=Unity;MATE;XFCE;Budgie:GNOME;
 NoDisplay=true
 StartupNotify=false
-- 
2.34.1

